test/gentlemanjerry.bats

```diff
@@ -149,3 +178,28 @@ teardown() {
   run timeout 10 java -cp /tmp/test SslTest api.logentries.com 25414
   [ "$status" -eq 0 ]
 }
+
+@test "Gentleman Jerry can connect to a TLSv1.2-only Endpoint" {
+  wait_for_tls12_server
+  run timeout 5 openssl s_client -CAfile /tmp/certs/jerry.crt -connect localhost:4433
+  [[ "$output" =~ "Verify return code: 0 (ok)" ]]
+
+  # Import cert into test truststore
+  keytool -importcert -file /tmp/certs/jerry.crt -keystore /tmp/certs/jerry.jks -storepass changeit -noprompt
```

Comment:
(nit) Not a huge fan of using `changeit` as the keystore password, since it could look like we forgot to consider what this param does (since it's always the default in guides). Something like `testpass` feels a little more appropriate?